data/core/macros/movetypes.cfg

```diff
@@ -0,0 +1,132 @@
+# a special macro to define slightly less nimble elves such
+# as fighters, who don't do quite so well in forest
+#define LESS_NIMBLE_ELF
+    [defense]
+        forest=40
+    [/defense]
+#enddef
+
+#define RACIAL_NOTES_ORCS_AND_GOBLINS
```

Comment:
This one probably doesn't need to move out of units.cfg. However, if it does move here then the file needs a `#textdomain wesnoth-help`.

Reply:
Is there a preference either way? If not, I'll move it back.
@ProditorMagnus as well.

Reply:
Not used in ageless, does not matter for me.

Reply:
Alright, I've moved `RACIAL_NOTES_ORCS_AND_GOBLINS` back into `data/core/units.cfg` in that case.